common/main/ks_Deva.xml

```diff
@@ -293,7 +293,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 							<day type="tue">बुवार</day>
 							<day type="wed">↑↑↑</day>
 							<day type="thu">↑↑↑</day>
-							<day type="fri">जुम्मा</day>
+							<day type="thu">↑↑↑</day>
 							<day type="sat">↑↑↑</day>
```

Comment:
is there a logical-group test applicable here?

Reply:
duplicate thu here!

Reply:
no, the vetters actually shortened the long one so it's shorter than this.  THIS one was a "copy inheritance to hard values" 1-off that we did in xml.   So i'm just dropping this one so that the shorter value from the vetters is inherited.